package.json

```diff
@@ -17,8 +17,7 @@
     "check:deps": "pnpm --recursive --parallel exec depcheck",
     "check:format": "prettier . --check",
     "check:lint": "turbo run lint --continue -- --quiet",
-    "check:react-exhaustive-deps": "turbo run lint --continue -- --quiet --rule 'react-hooks/exhaustive-deps: [error, {additionalHooks: \"(useMemoObservable|useObservableCallback)\"}]'",
```

Comment:
This check isn't needed anymore as `check:react-compiler` covers that and more, and is also run on the CI